projects/plugins/jetpack/extensions/blocks/story/player/components/overlay.js

```diff
@@ -28,6 +28,7 @@ export default function Overlay( {
 	slideCount,
 	showSlideCount,
 } ) {
+	const isRTL = document.documentElement.dir === 'rtl';
```

Comment:
I wonder if it would be better to rely on the `@wordpress/i18n` package, it includes `isRTL`, that may be just what we need:

`import { isRTL } from '@wordpress/i18n';`

Reply:
👍 That makes sense, we're already depending on `@wordpress/i18n` for translation